osu.Game/Rulesets/UI/Scrolling/DrawableScrollingRuleset.cs

```diff
@@ -131,7 +131,7 @@ private void load()
                     if (duration > maxDuration)
                     {
                         maxDuration = duration;
-                        baseBeatLength = timingPoints[i].BeatLength;
+                        baseBeatLength = timingPoints[i].BeatLength / Beatmap.BeatmapInfo.BaseDifficulty.SliderMultiplier;
```

Comment:
isn't it saner to just remove this from line line 160 below, rather than doing this? tested and seems to work correctly and pass tests.

Reply:
But we do want this no? As we discussed yesterday it's the base beatmap speed. Timing point velocity is multiplied into this inside `MultiplierControlPoint`.

Reply:
Will add a comment + test.